### PR TITLE
feat: implement landing page component

### LINKED ISSUE #4: Create Landing Page Component
**Is your feature request related to a problem? Please describe.**
Create a landing page for unauthenticated users.

**Describe the solution you'd like**
A landing page should have a login and register button to enable guests to create account.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by Yasholma**
## Description

- Created header component, footer component and home component
- Created unit tests for each of the components
- Installed bootstrap for responsive styling
- Added some rules to eslintrc.js config

Delivers #4 

## How Has This Been Tested?

1. In your terminal, run git clone https://github.com/BuildForSDG/SmeVest-Frontend.git
2. Run cd SmeVest-Frontend
3. Run git fetch origin ft-landing-page
4. Run git checkout origin/ft-landing-page
5. Run yarn install
6. Run yarn start and navigate to http://localhost:1234 on your browser
7. Run yarn test for unit tests

## Screenshots (if applicable, else remove this line / section)
image

image

image

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added necessary inline code documentation
- [x] I have added tests that prove my fix is effective and that this feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by achingachris — COMMENTED** [review]
honestly never worked with react, but I
Screenshot from 2020-05-11 22-57-42
 tried the project and got a building error, either I didn't get the procedure right or some config ain't right

**Comment by Yasholma** [comment]
> honestly never worked with react, but I
> Screenshot from 2020-05-11 22-57-42
> tried the project and got a building error, either I didn't get the procedure right or some config ain't right

It has been fixed. eslint was giving me issues. Sorry about that. Just clone the repo again.

**Review by Johnny2020Ade — COMMENTED** [review]
Good work

**Review by Johnny2020Ade — DISMISSED** [review]

**Comment by achingachris** [comment]
> > honestly never worked with react, but I
> > Screenshot from 2020-05-11 22-57-42
> > tried the project and got a building error, either I didn't get the procedure right or some config ain't right
> 
> It has been fixed. eslint was giving me issues. Sorry about that. Just clone the repo again.

okay, let me redo

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+              <Instagram />
+            </a>
+            <a href="https://twitter.com/yasholma" rel="noopener noreferrer" target="_blank">
+              <LinkedIn />
+            </a>
+          </ul>
+        </Col>
+      </Row>
+    </Container>
+    <p className="Copyright">&copy; Copyright 2020 SmeVest. All rights reserved. </p>
```
Comment (oyewoas) [inline src/components/Footer/Footer.js]: Welldone @Yasholma  this is nice, however i am wondering if we can set the year in such a way that i updates automatically, you can have a constant like `const date = new Date();` declared, then replace **2020** with `{date.getFullYear()}` in the `Footer Component`

**Review by oyewoas — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (Yasholma) [reply src/components/Footer/Footer.js]: Alright, I will implement that. Should I do it now or during the next PR?

Comment (oyewoas) [reply src/components/Footer/Footer.js]: I think you should do that now once and for all since its nothing much

Comment (Yasholma) [reply src/components/Footer/Footer.js]: Ok, I just did now.

**Review by oyewoas — APPROVED** [review]

**Review by Alasco95 — APPROVED** [review]
Excellent work

**Review by Johnny2020Ade — APPROVED** [review]

**Pull request merged**